justfile

```diff
@@ -96,11 +96,19 @@ configure-test-func:
   cmake -B $PWD/build/test-func -S $PWD \
     -DCMAKE_BUILD_TYPE=Debug \
     -DCMAKE_C_COMPILER=$C_COMPILER \
-    -DCMAKE_C_FLAGS="-std=c99 -Wno-error " \
+    -DCMAKE_C_FLAGS="-std=c99 -Wno-error" \
     -DATSDK_BUILD_TESTS="func" \
+    -DATSDK_BUILD_UNIT_TESTS=OFF \
+    -DATSDK_BUILD_FUNCTIONAL_TESTS=ON \
```

Comment:
These are no longer necessary. `-DATSDK_BUILD_TESTS="func"` is the correct way now.

The value can either be ON, OFF, "func" or "unit"

Reply:
Done in https://github.com/atsign-foundation/at_c/pull/490/commits/cb4ce8ceef8ec2135f5174749f41de179f8a25e4